include/ruby/internal/interpreter.h

```diff
@@ -141,7 +141,7 @@ void ruby_show_copyright(void);
  *
  * @param[in]  addr  A pointer somewhere on the stack, near its bottom.
  */
-void ruby_init_stack(volatile VALUE *addr);
+void ruby_init_stack(volatile void *addr);
```

Comment:
This is a public header; changing `VALUE` to `void` is not a breaking change I think (and where this is called inside ruby, it's called with a `(void *)` cast anyway today). However, removing `volatile` _would_ break a program which calls this with a volatile variable I think.

It doesn't _need_ to be volatile (we never dereference this address anyway), but it doesn't do any harm I suppose.